libbeat/_meta/config.reference.yml

```diff
@@ -610,9 +610,12 @@ output.elasticsearch:
 #setup.template.enabled: true
 
 # Template name. By default the template name is beatname.
-# The version of the beat will always be appended to the given name
-# so the final name is beatname-%{[beat.version]}.
-#setup.template.name: "beatname"
+# The f
```

Comment:
Ignore this doc and below for the moment. Take a look at the PR description on the behaviour.